airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -103,10 +102,10 @@ public static void attemptSQLCreateAndDropTableOperations(final String outputSch
       sqlOps.dropTableIfExists(database, outputSchema, outputTableName);
     } catch (final SQLException e) {
       if (Objects.isNull(e.getCause()) || !(e.getCause() instanceof SQLException)) {
-        throw new ConnectionErrorException(e.getSQLState(), e.getErrorCode(), e.getMessage(), e);
+        throw new ConfigErrorException(DisplayErrorMessage.getErrorMessage(e.getSQLState(), e.getErrorCode(), e.getMessage(), e), e);
```

Comment:
This is a good example of one that should be caught lower down (like you are doing) if we do it like this. becasue we know for sure the SQL that's happening. The error message could probably be better. SOmething more instructive for the user ("unable to create table in your schema X") or something.

If we would do this high up (in the main method), then it would be more nuanced about what kind of SQL error it was.